news/2018-07-16-cwc-2018-recap-grand-finals.md

```diff
@@ -0,0 +1,61 @@
+
+Rampant 6-1 victories over the Netherlands and Hong Kong, a 7-1 victory over 2016 runners-up Indonesia, and even a 7-3 victory over three-time Champions South Korea headlined their intent at both revenge, and consequently, a third osu!catch World Cup win of their own.
+
+A comfortable opening victory on [Memme - Goat Fantasia](https://osu.ppy.sh/beatmapsets/634496#fruits/1346273) would not provide much solace for the Chileans, as the Chinese team strung together five victories on the bounce, including a humiliating triple fail capitulation for Chile early into the rarely-seen DoubleTime pick, [nanobii - HYPERDRIVE](https://osu.ppy.sh/beatmapsets/639991#fruits/1357221).
+
+Chile would secure a second point on [Morimori Atsushi - Paradigm Shift](https://osu.ppy.sh/beatmapsets/727816#fruits/1536733) courtesy of a typically gutsy one-miss from eldnl, but by this point the damage was done. China had secured victories in each of the four mod pools and left Chile nowhere to turn, eventually securing a 7-2 victory, as well as a rematch against the Americans for the ultimate prize.
+
+![](/wiki/shared/news/2018-07-15-cwc-2018-recap-grand-finals/CNvCH_game9_720w.jpg)
+
+China contesting for their third World Cup triumph after 2012 and 2016, the United States making their first ever Grand Final appearance. The final showdown between these two teams, widely tipped for the Grand Final before and during the tournament, was about to begin. Having come from the Losers' Bracket, China would have to win two best-of-13 matches, back-to-back to take home the crown.
+
+The United States got off to the perfect start, winning the roll by a single point and settling the nerves with an opening pick victory on the punishing, old-style tones of [YuFu - Holy Moon](https://osu.ppy.sh/beatmapsets/34729#fruits/112852). China would immediately respond with a crushing DoubleTime victory as they have done so often throughout this tournament, but then the tactics begun.
+
+United States' captain Zak sought to exploit this by picking [Memme - Goat Fantasia](https://osu.ppy.sh/beatmapsets/634496#fruits/1346273), forcing China not only on to a pick they had shown to be weak on, but also to immediately adapt from AR 10.5 to AR 8 Hidden. This was a risky strategy that unforunately unsettled his own team more than the Chinese, who were allowed to obtain a narrow break-point victory and set the Americans on the back foot.
+
+The teams traded blows for a time, China coming out the victors in DoubleTime and HardRock, the United States on NoMod. After an enthralling eleven thousand point victory on [Morimori Atsushi - Paradigm Shift](https://osu.ppy.sh/beatmapsets/727816#fruits/1536733) where one player from both teams failed and US player chickenbible revived on the final note, the scores were perfectly poised at 5-3 to the Chinese.
```

Comment:
same deal here with chickenbible, link to profile on first mention.

Reply:
✅